README.md

```diff
@@ -23,16 +23,34 @@ The URL from which SonarQube will be downloaded, and the resulting directory nam
 
     sonar_mysql_username: sonar
     sonar_mysql_password: sonar
-    
```

Comment:
These spaces are intentional, so GitHub/other markdown parsers keep that entire section as one code block.